src/component/Todo/todo.css

```diff
@@ -0,0 +1,70 @@
+.todo-wrapper {
+  /* backdrop-filter: blur(10px); */
+  position: absolute;
+  left: 10px;
+  padding: 2rem;
+  /* border-radius: 10px; */
+  width: 10ch;
+  max-height: 50vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.todo-heading {
+  text-align: center;
+  font-size: 2rem;
+  color: var(--COLOR-WHITE);
+  margin-bottom: 1rem;
+}
+
+.todo-input {
+  background: transparent;
+  border: none;
+  border-bottom: 1px solid var(--COLOR-WHITE);
+  outline: none;
+  font-size: 2rem;
+  width: 100%;
+  text-align: center;
+  color: var(--COLOR-WHITE);
+}
+
+.list-display {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+
+.list-display::-webkit-scrollbar {
+  display: none;
+}
+
+.list-display label {
+  cursor: pointer;
+  font-size: 2rem;
+  overflow-wrap: break-word;
+  color: var(--COLOR-WHITE);
+}
+
+.list-display input {
+  /* --size: 1.75rem;
+  width: var(--size);
+  height: var(--size); */
```

Comment:
Suggestion: You could remove the commented code

Reply:
done